_build_cfg.yml

```diff
@@ -83,8 +83,8 @@ Topics:
 Name: Using OpenShift
 Dir: using_openshift
 Topics:
-  - Name: Overview
-    File: overview
+#  - Name: Overview
+#    File: overview
```

Comment:
@tpoitras @adellape The 'Overview' discussion we had was for each individual topic, NOT for topic directories. We need the Overview topics within each topic directory for various reasons, so we should NOT remove them.